back/engines/commercial/analysis/spec/acceptance/tags_spec.rb

```diff
@@ -38,16 +38,47 @@
+      parameter :'author_custom_<uuid>_to', 'Filter by custom field value of the author for numerical or date fields, smaller than or equal to. Replace <uuid> with the custom_field id'
+      parameter :'author_custom_<uuid>', 'Filter by custom field value of the author, for select, multiselect, date and number fields (union). Replace <uuid> with the custom_field id', type: :array
+      parameter :published_at_from, 'Filter by input publication date, after or equal to', type: :date
+      parameter :published_at_to, 'Filter by input publication date, before or equal to', type: :date
+      parameter :reactions_from, 'Filter by number of reactions on the input, larger than or equal to', type: :integer
+      parameter :reactions_to, 'Filter by number of reactions on the input, smaller than or equal to', type: :integer
+      parameter :votes_from, 'Filter by number of votes on the input, larger than or equal to', type: :integer
+      parameter :votes_to, 'Filter by number of votes on the input, smaller than or equal to', type: :integer
+      parameter :comments_from, 'Filter by number of comments on the input, larger than or equal to', type: :integer
+      parameter :comments_to, 'Filter by number of comments on the input, smaller than or equal to', type: :integer
+    end
+
     let(:analysis) { create(:analysis) }
     let(:analysis_id) { analysis.id }
+    let(:comments_from) { 5 }
     let!(:tags) { create_list(:tag, 3, analysis: analysis) }
+    let!(:input1) { create(:idea, project: analysis.project, comments_count: 2) }
+    let!(:input2) { create(:idea, project: analysis.project, comments_count: 5) }
+    let!(:input3) { create(:idea, project: analysis.project, comments_count: 5) }
+    let!(:taggings) do
+      create(:tagging, input: input1, tag: tags[0])
+      create(:tagging, input: input1, tag: tags[1])
+      create(:tagging, input: input2, tag: tags[1])
+    end
 
     context 'when admin' do
       before { admin_header_token }
 
       example_request 'lists all tags of an analysis' do
         assert_status 200
-        expect(json_response_body[:data].pluck(:id)).to match_array(tags.pluck(:id))
+        expect(response_data.pluck(:id)).to match_array(tags.pluck(:id))
+        expect(response_data.find { |d| d[:id] == tags[0].id }[:attributes][:total_input_count]).to eq 1
+        expect(response_data.find { |d| d[:id] == tags[1].id }[:attributes][:total_input_count]).to eq 2
+        expect(response_data.find { |d| d[:id] == tags[2].id }[:attributes][:total_input_count]).to eq 0
+
+        expect(response_data.find { |d| d[:id] == tags[0].id }[:attributes][:filtered_input_count]).to eq 0
+        expect(response_data.find { |d| d[:id] == tags[1].id }[:attributes][:filtered_input_count]).to eq 1
+        expect(response_data.find { |d| d[:id] == tags[2].id }[:attributes][:filtered_input_count]).to eq 0
```

Comment:
I think these tests are reflecting the filtering due to `comments_from: 5`, correct?

It took me a while to figure that out. I wonder if this could be made clearer somehow? Maybe separate them into their own example, and make the example name describe this filtering? E.g:
```Ruby
example_request 'lists filtered tags of analysis (this example; comments_from: 5)' do
```
... or something like that?

Reply:
You're right, will adapt